src/Symfony/Bridge/Monolog/Formatter/ConsoleFormatter.php

```diff
@@ -11,45 +11,191 @@
 
 namespace Symfony\Bridge\Monolog\Formatter;
 
-use Monolog\Formatter\LineFormatter;
-use Monolog\Logger;
+use Monolog\Formatter\FormatterInterface;
+use Symfony\Component\VarDumper\Cloner\Data;
+use Symfony\Component\VarDumper\Cloner\Stub;
+use Symfony\Component\VarDumper\Cloner\VarCloner;
+use Symfony\Component\VarDumper\Dumper\CliDumper;
 
 /**
  * Formats incoming records for console output by coloring them depending on log level.
  *
  * @author Tobias Schultze <http://tobion.de>
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
  */
-class ConsoleFormatter extends LineFormatter
+class ConsoleFormatter implements FormatterInterface
 {
-    const SIMPLE_FORMAT = "%start_tag%[%datetime%] %channel%.%level_name%:%end_tag% %message% %context% %extra%\n";
+    const SIMPLE_FORMAT = "%datetime% %start_tag%%level_name%%end_tag% <comment>[%channel%]</> %message%%context%%extra%\n";
+    const SIMPLE_DATE = 'H:i:s';
+
+    private static $levelColorMap = array(
+        100 => 'fg=white',
```

Comment:
1. because the map will not be aligned ;) just kidding !
2. PSR contant with int level does not exist


I updated the PR to use Monolog\Logger::CONSTANT